### PR TITLE
rename internal terms

### PR DESCRIPTION
**Pull request by wochinge**
followup of https://github.com/RasaHQ/rasa/issues/6028
- we already use the `main` branch, but I updated some internal vocabulary as well

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Continuous Integration
 on:
   push:
     branches:
-    - master
+    - main
```
Comment (wochinge) [inline .github/workflows/ci.yml:5]: oopsie. I think that never ran 😆

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -2,14 +2,14 @@ name: Continuous Integration
...
     tags:
     - '*'
   pull_request:
 
 env:
-  RASA_X_DEMO_VERSION: '0.33.0'
-  RASA_IMAGE_TAG: '2.1.3-full'
+  RASA_X_DEMO_VERSION: '0.35.0'
```
Comment (wochinge) [inline .github/workflows/ci.yml:11]: had to bump this as it was not part of the page size anymore (increasing the page size didn't work). I think we need a better approach for this, what do you think?

Comment (wochinge) [reply .github/workflows/ci.yml:11]: seems it's not used as the test runs against `main`?

Comment (tczekajlo) [reply .github/workflows/ci.yml:11]: > had to bump this as it was not part of the page size anymore (increasing the page size didn't work). I think we need a better approach for this, what do you think?

Yes, I'll take care of it, We can use the Manifest API to check a tag.

It should run on a PR

File: .github/workflows/ci.yml (hunk 1 above)
Comment (tczekajlo) [reply .github/workflows/ci.yml:5]: It should run on a PR though

Comment (wochinge) [reply .github/workflows/ci.yml:5]: yep, just not after merges 👍🏻

File: .github/workflows/ci.yml (hunk 2 above)
Comment (wochinge) [reply .github/workflows/ci.yml:11]: Mhm, weird. Then I don't understand why the CI keeps failing with the old version 🤔 But it's also late 🙈

**Comment by wochinge** [comment]
Thanks for your help @tczekajlo 🙌🏻

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -69,14 +69,14 @@ jobs:
...
-        wget https://github.com/RasaHQ/rasa-x-demo/blob/0.33.0/models/model.tar.gz?raw=true \
+        wget https://github.com/RasaHQ/rasa-x-demo/blob/${{ env.RASA_X_DEMO_VERSION }}/models/model.tar.gz?raw=true \
           -O test_model.tar.gz
 
     - id: action
       name: Rasa Test & Train - Rasa X Demo
       uses: ./
       with:
-        rasa_version: '2.0.0-full'
+        rasa_version: ${{ env.RASA_IMAGE_TAG }}
```
Comment (wochinge) [inline .github/workflows/ci.yml:79]: weird that this didn't show up in my global search 🤔

**Review by tczekajlo — APPROVED** [review]
@wochinge looks like everything is green here 😄

**Pull request merged**